src/test/java/com/microsoft/sqlserver/jdbc/datatypes/SQLServerSpatialDatatypeTest.java

```diff
@@ -187,6 +187,7 @@ public void testFullGlobeWkb() throws SQLException {
     }
 
     @Test
+    @Tag("xAzureSQLDW")
```

Comment:
Why don't we skip the entire class instead?

Reply:
Not all tests are skipped in this class.

Reply:
Spatial datatypes are not supported on DW, I thought we could skip the entire class for that reason, instead of tagging so many tests. Looks like the tests that are not skipped do not contact database at all. I will leave this up to you.